tests/test_text_data.py

```diff
@@ -12,3 +27,52 @@ def test_should_load_backwards_lm():
 
     as_text = [text_ds.vocab.itos[x] for x in batch[0]]
     np.testing.assert_array_equal(as_text[:2], ["world", "hello"])
+
+def test_from_csv():
+    path = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'data', 'tmp')
+    os.makedirs(path)
+    filename = 'text'
+    filepath = os.path.join(path, filename+'.csv')
+    try:
+        text_csv_file(filepath)
+        data = TextClasDataBunch.from_csv(path, train=filename, valid=filename)
+        assert len(data.classes) == 2
+        assert set(data.classes) == set([True, False])
+    finally:
+        shutil.rmtree(path)
+
+def test_from_csv_multilabel():
```

Comment:
similar, could combine this with `test_from_csv()` into a more generic form (as they do share common code) - but again have kept it separate for clarity instead of an if-else inside 1 longer function